common/lib/dependabot/dependency.rb

```diff
@@ -53,7 +55,8 @@ def to_h
         "requirements" => requirements,
         "previous_version" => previous_version,
         "previous_requirements" => previous_requirements,
-        "package_manager" => package_manager
+        "package_manager" => package_manager,
+        "metadata" => metadata
       }
```

Comment:
Might want to call `compact` here, so that setups without metadata don't include that key.